src/graphql/main/docs/README.md

```diff
@@ -0,0 +1,48 @@
+# Lightning Integration
+
+## Overview
+The API endpoints are
+
+testnet: `https://api.testnet.galoy.io/graphql`
+
+mainnet: `https://api.mainnet.galoy.io/graphql`
```

Comment:
I think we should only list the `testnet` one for now. it will be renamed to `staging` btw.

the `mainnet` will soon be very dependent to the deployment, so there will be many and their endpoints won't necessarily be public.